packages/client/hmi-client/src/main.ts

```diff
@@ -10,6 +11,7 @@ const app = createApp(App);
 app.use(createPinia());
 app.use(router);
 app.use(PrimeVue, { ripple: true });
+app.use(createLogger({}));
```

Comment:
This looks a little strange, is the usage correct? I would have expected that createLogger to be called only once but we are calling it twice?

We have
```
import { createLogger } from 'vue-logger-plugin';
app.use(createLogger({}));
```

shouldn't this be 

```
import { logger } from '@/utils/logger-something-something';
app.use(logger);
```

where the logger instance is created in the util?

Reply:
no you're right.  found it strange that this worked in the first place.